src/Tools/Github/GitMergeBot/project.json

```diff
@@ -10,6 +10,6 @@
     "net46": { }
   },
   "runtimes": {
-    "win7": { }
+    "win": { }
```

Comment:
This is going to need to be 'win7' when we do the further work of making this project use the Roslyn props/targets files